src/component/Footer/Footer.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+
+interface FooterProps {
+  todos: Todo[]
+  filteredBy: string;
+  setFilteredBy: (option: string) => void;
+}
+export const Footer: React.FC<FooterProps> = ({
+  todos,
+  filteredBy,
+  setFilteredBy,
+}) => {
+  const todoCount = todos.filter((todo) => !todo.completed).length;
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        {`${todoCount} items left`}
```

Comment:
check logic to count not completed todos
it this case it should be always 3
![Screenshot from 2023-06-03 16-40-51](https://github.com/mate-academy/react_todo-app-loading-todos/assets/33029864/902ceeb5-51b5-4b26-b62b-ffb6a685df40)